privacy-protections/click-to-load/main.js

```diff
@@ -86,6 +90,11 @@ function downloadTheResults () {
 const downloadButton = document.querySelector('#download');
 downloadButton.addEventListener('click', () => downloadTheResults());
 
-window.onload = function () {
+const updateLoads = function () {
     document.getElementById('facebook_call_count').innerHTML = facebookCalls ? '<span style="color:RED;">DETECTED</span>' : '<span style="color:GREEN;">NONE</span>';
```

Comment:
I think using `innerHTML` is kind of frowned upon. How about having the span element there already, but with an ID, and then setting the span's class/style + text here?

Reply:
innerHTML is crude but effective, no issue in this case, its not like we have a bunch of event listeners hanging off that.  I'd rather KISS